controller/array_action/array_mediator_ds8k.py

```diff
@@ -643,3 +676,11 @@ def _generate_snapshot_response(self, api_snapshot, source_volume_name):
     def validate_flashcopy(self, flashcopy_id):
         api_flashcopy = self._get_flashcopy(flashcopy_id)
         return api_flashcopy.state == 'valid'
+
+    def _check_snapshot_use_status(self, snapshot_id, flashcopy_list):
+        for flashcopy in flashcopy_list:
+            logger.info("Deleting flashcopy: {}".format(flashcopy))
+            if flashcopy.sourcevolume == snapshot_id:
+                flashcopy_rel = self._get_flashcopy(flashcopy.id)
```

Comment:
what is the difference between `flashcopy` and `flashcopy_rel` here?

Reply:
answered in: https://github.com/IBM/ibm-block-csi-driver/pull/200#discussion_r451531608